runtime/near-vm-logic/src/gas_counter.rs

```diff
@@ -64,29 +75,81 @@ impl GasCounter {
 
     #[cfg(feature = "costs_counting")]
     #[inline]
-    fn inc_ext_costs_counter(&self, cost: ExtCosts, value: u64) {
+    fn inc_ext_costs_counter(&mut self, cost: ExtCosts, value: u64) {
         EXT_COSTS_COUNTER.with(|f| {
```

Comment:
I thought you suggested to remove `EXT_COSTS_COUNTER` because it is an expensive thread-local tool. If we have `profile` that it looks like it should be easy to remove `EXT_COSTS_COUNTER`

Reply:
Well, they have different semantics with profile (profile sums costs, EXT_COSTS_COUNTER counts invocations) and EXT_COSTS_COUNTER id being used in the cost estimator, so I decided to keep it.